test_py/test_client.py

```diff
@@ -0,0 +1,186 @@
+import pytest
+from src_py.grndwork_api_client.access_tokens import get_access_token
+from src_py.grndwork_api_client.client import Client
+from src_py.grndwork_api_client.config import DATA_URL
+from src_py.grndwork_api_client.dtos import DataFile, DataFileHeaders, Station
+from src_py.grndwork_api_client.dtos import GetDataQuery, GetStationsQuery
+from src_py.grndwork_api_client.make_request import ContentRange
+from src_py.grndwork_api_client.make_request import make_paginated_request
+from src_py.grndwork_api_client.make_request import make_request
+
+
+def describe_client():
+
+    def get_datafile():
+        headers = DataFileHeaders(
+            meta={},
+            columns=[],
+            units=[],
+            processing=[],
+        )
+        data_file = DataFile(
+            source='src',
+            filename='filename.dat',
+            is_stale=False,
+            headers=headers,
+            records=[],
+        )
+        return data_file
+
+    def get_station():
+        station = Station(
+            client_uuid='client_uuid',
+            client_full_name='full_name',
+            client_short_name='short_name',
+            site_uuid='station_uuid',
+            site_full_name='site_full_name',
+            station_uuid='station_uuid',
+            station_full_name='station',
+            description='',
+            latitude=0,
+            longitude=0,
+            altitude=0,
+            timezone_offset=1,
+            start_timestamp='1',
+            end_timestamp='1',
+            data_file_prefix='PRE',
+            data_files=[],
+        )
+        return station
+
+    @pytest.fixture(name='get_access_token', autouse=False)
```

Comment:
Seems like this could be set to `autouse=True`, then you don't need to explicitly include it in the args below?